python/interpret_community/explanation/explanation.py

```diff
@@ -494,7 +494,10 @@ def _perf_dict(self, y, y_hat, i):
         di = {}
         di["actual"] = y[i]
         di["predicted"] = y_hat[i]
-        di["residual"] = y[i] - y_hat[i]
+        if isinstance(y[i], str) or isinstance(y_hat, str):
```

Comment:
I think there was another perf dict fix a few weeks ago. Might be an indicator that unit tests are important here